src/index.js

```diff
@@ -1,10 +1,13 @@
 import './style.css';
 import addTask from './modules/addTask.js';
 import removeTask from './modules/removeTask.js';
+import removeCompleted from './modules/removeCompleted.js';
+import trashCompleted from './modules/completed.js';
 import { saveStorage, getStorage } from './modules/storage.js';
```

Comment:
- [ ] The localStorage functionality is not working as intended.  Let me explain with a video:

https://share.getcloudapp.com/DOum7J4R

The items are not being updated and saved correctly in `localStorage`, upon deletion of some of them.
Please, check your `localStorage `related algorithms to achieve the correct functionality.

Reply:
Done